src/main/java/Duke.java

```diff
@@ -1,10 +1,58 @@
+import java.util.Scanner;
+//so scanner class can be used
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+    public static void printLine() {
+        System.out.println("________________________________________");
+    }
+    public static void printShortLine() {
+        System.out.println("_____________");
+    }
+    public static void printHello() {
+        String logo =
+                "__   __   ___    _____ \n"
+                        + "\\ \\ / /  / _ \\  | ___ |\n"
+                        + " \\ Y /  | | | |     | | \n"
+                        + "  \\ /   | | | |     | | \n"
+                        + "  | |   | |_| |  ___| | \n"
+                        + "  |_|    \\___/  |____/          \n";
+        printLine();
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm YOJ");
+        System.out.println("What can I do for you?");
+        printLine();
+    }
+
+    public static void main(String[] args) {
+        printHello();
+        // get user input
+        Scanner in = new Scanner(System.in);
+        String userInput;
+        userInput = in.nextLine();
+
+        while(!userInput.equals("bye")) {
+            printShortLine();
+            System.out.println("added: " + userInput);
+            printLine();
+            if (userInput.equals("list")) {
+                Task.printList();
+            } else if(userInput.matches("mark \\d+")) {
```

Comment:
may want to set constants to identify the regex expression, as regex may not be intuitively readable